frontend/src/components/common/AutoCompleteInput/__snapshots__/AutoCompleteInput.test.tsx.snap

```diff
@@ -0,0 +1,72 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
```

Comment:
I have developed a fear of snapshot tests lately, and would prefer to not use them at all. However we can discuss this f2f and hopefully I can explain why 😁

Reply:
I see, I also had my doubts, let's discuss this further f2f! 👍🏼